extra/1-radio-stations.js

```diff
@@ -14,17 +14,56 @@
  */
 
 // `getAllFrequencies` goes here
+function getAllFrequencies() {
+  let array = [
+    87,
+    88,
+    89,
+    90,
+    91,
+    92,
+    93,
+    94,
+    95,
+    96,
+    97,
+    98,
+    99,
+    100,
+    101,
+    102,
+    103,
+    104,
+    105,
+    106,
+    107,
+    108,
+  ];
+  return array;
+}
 
 /**
  * Next, let's write a function that gives us only the frequencies that are radio stations.
  * Call this function `getStations`.
  *
+ *
  * This function should:
  * - Get the available frequencies from `getAllFrequencies`
  * - There is a helper function called isRadioFrequency that takes an integer as an argument and returns a boolean.
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
+function getStations() {
```

Comment:
So on this question I doubt this is what they wanted hence I high-key believe this may not be passing all test cases...

### Here's an idea behind this function

1. You have created a function **getAllFrequencies** which returns an array back to you
2. Now what you need from this function called **getStations** you need to get **available frequencies from getAllFrequencies function that you created**
3. Now you can call the **getAllFrequencies** function save the returned array in a variable resulting to you having an array of all frequencies which you returned in that function **getAllFrequencies** 
4. Now they are telling us in the instructions that they made a helper function called **isRadioFrequency** now this function will tell **true** if the frequency is a radio station or **false** if not 
5. Now you have interact with that helper function to help you clean up the array containing all **frequencies** to only remain with frequencies that are for a radio station
6. Finally return all **radio stations**

## Hope this helps 🔥💪

Reply:
Thanks Ntshembo my code works fine now.